manifests/c/CABS/SalesPrix/6.15.0.0/CABS.SalesPrix.locale.de-DE.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: CABS.SalesPrix
+PackageVersion: 6.15.0.0
+PackageLocale: de-DE
+Publisher: CAB+S GmbH
+PackageName: SalesPrix

```

Comment:
```suggestion
PackageName: SalesPrix
PackageUrl: https://www.scanprix.de/
```

Reply:
Hello
i don't understand what you are saying. I did what wingetcreate tells me to do. The name of the program is "SalesPrix", the name of the company is "CAB+S GmbH", in short: CABS.
What is the difference between "PackageIdentifier" and "PackageName"?
Please explain.
Best regards
Klaus

Reply:
Hi @KlausGue,

> i don't understand what you are saying. 

I'm suggesting to add a PackageUrl that points at the application's website. This helps satisfy the requirement that it's possible to navigate from the PackageUrl to the InstallerUrl. Sorry for being obtuse. 

> I did what wingetcreate tells me to do. 

This isn't a required field, so WingetCreate might not ask for it. 

> The name of the program is "SalesPrix", the name of the company is "CAB+S GmbH", in short: CABS.

Yes, the PackageIdentifier is fine, which is why I didn't suggest any changes to it. 

> What is the difference between "PackageIdentifier" and "PackageName"?

Here's the `defaultLocale` entry for our [1.4.0 Schema](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.4.0/defaultLocale.md).
- PackageIdentifier is a unique way of identifying the package across the WinGet package manager's systems and components. 
- PackageName is used in displaying the package locally, and should match the Apps or Add and Remove Programs name. 

Does this help? The PR is well-formed and should merge without further changes.